x-pack/plugins/security_solution/public/cases/containers/use_post_push_to_service.tsx

```diff
@@ -202,6 +195,7 @@ export const formatServiceRequestData = (
     description,
     externalId: actualExternalService?.externalId ?? null,
     title,
+    ...(connector.fields ?? {}),
```

Comment:
this is not making sense to me with the return type of the function, I do not see the fields attribute in `ServiceConnectorCaseParams`. Can you explain to me?

Reply:
There is no need for the fields to be there. I am not sure how it went there tbh. I will remove it.

Reply:
I was wrong. We need to pass the fields to the external service. I opened this [issue](https://github.com/elastic/kibana/issues/79697) to track it.